signing/sawtooth_signing/core.py

```diff
@@ -128,6 +128,22 @@ def new_random_private_key(self):
             (:obj:`PrivateKey`): a random private key
         """
 
+    @abstractmethod
+    def private_key_from_bytes(self, bytes):
+        """Generates a new random PrivateKey using this context.
```

Comment:
Let's update these docstrings so they aren't just a copy/paste of `new_random_private_key`